### PR TITLE
fix: Reference official docs in README

### PR DESCRIPTION
**Pull request by stefannica**
The official docs have their own repo and website now, and a lot of
the information in README.md is duplicated there. To elliminate
duplication, the official docs are linked here instead.

### CODE REVIEW

**Comment by jsuchome** [comment]
This `+19 −121 ` is good, however now I miss the part where it mentions that _this_ repository basically contains the installer.

**Comment by stefannica** [comment]
> This `+19 −121 ` is good, however now I miss the part where it mentions that _this_ repository basically contains the installer.

ok, let me add something for that

File: README.md [hunk 1]
```diff
@@ -15,53 +15,30 @@ Build your own custom MLOps orchestration workflows from composable automation r
...
-
-The FuseML team is still in the process of defining the project strategy, but we can say with some certainty that the project will develop in the following main directions aligned with the stated principles:
+The project is under heavy development following the main directions:
 1. adding features and enhancements to improve flexibility and extensibility
-2. add support for more community shared AI/ML tools
-3. create more composable automation blocks adapted to the existing as well as new AI/ML tools
+2. adding support for more community shared AI/ML tools
+3. creating more composable automation blocks adapted to the existing as well as new AI/ML tools
+
+Take a look at [our Project Board](https://github.com/orgs/fuseml/projects/1) to see what we're working on and what's in store for the next release.
```
Comment (flaviodsr) [inline README.md:33]: With that it might make sense to remove line 9.

File: README.md [hunk 2]
```diff
@@ -15,53 +15,30 @@ Build your own custom MLOps orchestration workflows from composable automation r
...
 
 ## Basic Workflow
 
-The basic FuseML workflow can be described as a GitOps type of workflow that starts with your ML code and automatically runs all the steps necessary to build and serve your machine learning model. FuseML's job begins when your machine learning code is ready for execution.
+The basic FuseML workflow can be described as an MLOps type of workflow that starts with your ML code and automatically runs all the steps necessary to build and serve your machine learning model. FuseML's job begins when your machine learning code is ready for execution.
 
 1. install the FuseML Ochestrator in a kubernetes cluster of your choice (see [Installation Instructions](# Install))
```
Comment (flaviodsr) [inline README.md:39]: Maybe just `install FuseML in a kubernetes...`

Also the link to the installation instructions is missing.

File: README.md [hunk 3]
```diff
@@ -74,91 +51,12 @@ The basic FuseML workflow can be described as a GitOps type of workflow that sta
 
 ### Model Training
 
+* MLFlow
 * DeterminedAI (TBD)
 
 ### Model Serving and Monitoring
 
 * MLFLow
```
Comment (flaviodsr) [inline README.md:59]: MLFlow serving is not yet supported.

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -15,53 +15,30 @@ Build your own custom MLOps orchestration workflows from composable automation r
...
 
 ## Basic Workflow
 
-The basic FuseML workflow can be described as a GitOps type of workflow that starts with your ML code and automatically runs all the steps necessary to build and serve your machine learning model. FuseML's job begins when your machine learning code is ready for execution.
+The basic FuseML workflow can be described as an MLOps type of workflow that starts with your ML code and automatically runs all the steps necessary to build and serve your machine learning model. FuseML's job begins when your machine learning code is ready for execution.
 
 1. install the FuseML Ochestrator in a kubernetes cluster of your choice (see [Installation Instructions](# Install))
 2. write your code using the AI/ML library of your choice (e.g. TensorFlow, PyTorch, SKLearn, XGBoost)
-3. organize your code using one of the conventions or experiment tracking tools supported by FuseML
+3. organize your code using one of the [conventions and experiment tracking tools](# Experiment Tracking and Versioning) supported by FuseML
```
Comment (flaviodsr) [inline README.md:41]: the link for Experiment Tracking and Versioning is also missing.

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stefannica) [reply README.md:33]: good point

File: README.md (hunk 2 above)
Comment (stefannica) [reply README.md:39]: thanks, would've missed that

File: README.md (hunk 3 above)
Comment (stefannica) [reply README.md:59]: it was at some point :smile: I forgot

**Review by flaviodsr — APPROVED** [review]
LGTM

**Pull request merged**